src/components/select/select.js

```diff
@@ -776,6 +777,14 @@ function OptionDirective($mdButtonInkRipple, $mdUtil) {
     var optionCtrl = ctrls[0];
     var selectCtrl = ctrls[1];
 
+    if (selectCtrl.isMultiple) {
+      element.attr('md-checkbox-enabled', '');
+      if (!checkboxElement) {
+        checkboxElement = angular.element('<div class="_md-container"><div class="_md-icon"></div></div>');
```

Comment:
Ah, I just misread the code. I wouldn't worry too much about the lazy initializing here; the cost of creating the element once at start-up should be pretty small.